Cargo.toml

```diff
@@ -287,6 +287,7 @@ rsa_signing = []
 slow_tests = []
 test_logging = []
 use_heap = []
+bn_tests = []
```

Comment:
Yea you're right, that did seem like a breaking change to the intent at least, I'll fix it :)